Automattic/wp-calypso
# Re-encrypt circle config with new CONFIG_KEY

#### Changes proposed in this Pull Request

This PR re-encrypts `/test/e2e/config/encrypted.enc` with the new encryption key.

#### Testing instructions

* Grab the new E2E config decode key from secret store
* Run `CONFIG_KEY="" npm run decrypt-e2e-config`
* Confirm there is a new file `/test/e2e/config/local-.json` that you're able to view
